packages/fhicl-cpp/package.py

```diff
@@ -78,8 +78,11 @@ def setup_build_environment(self, env):
         sanitize_environments(env, "PATH")
 
     def setup_run_environment(self, env):
-        bindir = self.prefix.bin
+        env.prepend_path("PATH", self.prefix.bin)
+        env.prepend_path("ROOT_INCLUDE_PATH", self.prefix.include)
+        env.append_path("CET_PLUGIN_PATH", self.prefix.lib)
```

Comment:
- See earlier comment about `ROOT_INCLUDE_PATH`--should probably not come in until `canvas-root-io`
- `fhicl-cpp` provides no plugins, so `CET_PLUGIN_PATH` should not be adjusted here